internal/blueprint/customizations.go

```diff
@@ -64,6 +65,12 @@ type ServicesCustomization struct {
 	Disabled []string `json:"disabled,omitempty" toml:"disabled,omitempty"`
 }
 
+type FilesystemCustomization struct {
+	Name       *string `json:"name,omitempty" toml:"name,omitempty"`
```

Comment:
Not sure we need `name`? Is that meant to be used as file system label? I'd vote for deferring that, but happy to be proven wrong :)

Reply:
Oh yeah, hadn't pushed those changes, meant for it to be `label` but we can defer that

Reply:
Probably exposing the label makes sense, but we may want to review if there are any restrictions we should enforce/keep in mind before adding it.